.github/actions/get-new-env-vars-v1/src/run.ts

```diff
@@ -0,0 +1,45 @@
+import { getExecOutput } from '@actions/exec'
+import type { Core } from './types'
+
+export default async function run(core: Core) {
+  try {
+    const envFilePath = core
+      .getInput('env-file-path', { required: true })
+      .trim()
+    const head = (core.getInput('head') || 'release').toLowerCase().trim()
+    const base = (core.getInput('base') || 'main').toLowerCase().trim()
+
+    core.info(`Finding new env vars from ${head} to ${base}...`)
+
+    const {
+      stdout: envVars,
+      stderr: envVarsError,
+      exitCode: envVarsExitCode
+    } = await getExecOutput(
+      `git diff origin/${base}...origin/${head} -- ${envFilePath}`
+    )
+
+    if (envVarsExitCode) {
+      throw new Error(`Error getting env vars: \n${envVarsError}`)
```

Comment:
It may be useful to print `stderr` here 🤷

Reply:
`envVarsError` is `strderr` from the `getExecOutput` command 👀

Reply:
The entire run function is in a try/catch and the catch outputs the error message and sets the run to fail

Reply:
> envVarsError is strderr from the getExecOutput command 👀

Ah, you're right. When skimming this, I thought you were including the exit code (not stderr). 👍